lisa/tools/df.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import re
+from dataclasses import dataclass
+from typing import List, Optional
+
+from lisa.executable import Tool
+from lisa.util import find_patterns_in_lines
+
+
+@dataclass
+class PartitionInfo(Tool):
```

Comment:
It's good to have this class, and make sure it's reusable in lsblk, fdisk.

Reply:
I didn't get this comment. Can you please provide some more context on this?

Reply:
I see there is a same name class `PartitionInfo` in lsblk. They can be the same one. It doesn't need to define different PartitionInfo class in each tool.

Reply:
The field are different for both of these. Both df and lsblk give different kind of info for the disk

Reply:
How about we move ` get_partition_information(self)` to disk features and populate info from both lsblk and df?

Reply:
Disk is different with partitions. We can leave them in lsblk and df themselves. Their information is overlap. We can define a structure to include information from both of them, and fill in different information from different command.

Reply:
Will it be merged in next push? And it doesn't need to inherit from the `Tool`.